### PR TITLE
Allow nightly builds to fail until #1731 can be fixed.

### PR DESCRIPTION
**Pull request by jaraco**

### CODE REVIEW

**Comment by jaraco** [comment]
As you can see, CI is failing due to linters.

**Comment by webknjaz** [comment]
> CI is failing due to linters

No. The failing linter is allowed to fail, which allows the next stage to proceed with running jobs.

The real failures are:
```console
Results (137.32s):
     288 passed
       2 xpassed
       4 failed
         - cherrypy/test/test_httputil.py:62 test_invalid_status[hey-Illegal response status from server ('hey' is non-numeric).]
         - cherrypy/test/test_httputil.py:62 test_invalid_status[status_code1-Illegal response status from server ({'hey': 'hi'} is non-numeric).]
         - cherrypy/test/test_httputil.py:62 test_invalid_status[1-Illegal response status from server (1 is out of range).]
         - cherrypy/test/test_httputil.py:62 test_invalid_status[600-Illegal response status from server (600 is out of range).]
       3 xfailed
       2 skipped
```

File: .travis.yml [hunk 1]
```diff
@@ -85,6 +85,7 @@ jobs:
   # TODO: fix tests
   - python: *pypy3
   - env: TOXENV=pre-commit-pep257
+  - python: nightly
```
Comment (webknjaz) [inline .travis.yml:88]: This change works:
> Your build matrix was set to allow the failure of job 2075.7.

**Comment by webknjaz** [comment]
The last commit on `master` was green at the time of merging but after a few days of cron runs of builds against the same commit it became red:
:heavy_check_mark: https://travis-ci.org/cherrypy/cherrypy/builds/551539222 (last green)
:negative_squared_cross_mark: https://travis-ci.org/cherrypy/cherrypy/builds/552272394 (first red)

I believe, something has changed in the runtime of the CI. Which is a separate problem itself.

**Comment by webknjaz** [comment]
#### Root cause

`pytest` got upgraded from `4.6.3` to `5.0.0`

**Comment by webknjaz** [comment]
This should be fixed by https://github.com/cherrypy/cherrypy/commit/96b34dfea7853b0189bc0a3878b6ddff0d4e505c now.

**Comment by webknjaz** [comment]
Okay, https://github.com/cherrypy/cherrypy/commit/14c12d2420a4b3765bb241250bd186e93b2f25eb really fixes it.

**Pull request merged**